Tests/Request/GetInboxMessagesRequestBuilderTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+namespace HP\Bundle\EmailApiBundle\Tests\Request;
+
+use HP\Bundle\EmailApiBundle\Request\GetInboxMessagesRequestBuilder;
+
+class GetInboxMessagesRequestBuilderTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var GetInboxMessagesRequestBuilder
+     */
+    private $builder;
+
+    public function setUp()
+    {
+        $this->builder = new GetInboxMessagesRequestBuilder();
+    }
+
+    /**
+     * Tests the builder creating all the parameters available to set and verify them
+     */
+    public function testBuilder()
+    {
+        $maxResults = 100;
```

Comment:
Use as `const MAX_RESULTS = 100;` like in `Tests/Usecase/Inbox/GetInboxMessagesUsecaseTest.php`